### PR TITLE
feat(Worklets): react-native-worklets node package stub 

### PR DESCRIPTION
**Pull request by tjzel**
## Summary

I was doing some weird git stuff and github closed previous PR 🤷 #6790

## Test plan

🚀

### CODE REVIEW

File: packages/react-native-worklets/package.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
+  "name": "react-native-worklets",
+  "version": "0.0.1",
+  "description": "🚀",
```
Comment (piaskowyk) [inline packages/react-native-worklets/package.json:4]: This isn't a blocker for the merge, but we need to add a proper description here

File: packages/react-native-worklets/RNWorklets.podspec [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      s.dependency "RCTRequired"
+      s.dependency "RCTTypeSafety"
+      s.dependency "ReactCommon/turbomodule/core"
+    end
+  end
+  
+  s.pod_target_xcconfig = {
+    "USE_HEADERMAP" => "YES",
+    "DEFINES_MODULE" => "YES",
+    "HEADER_SEARCH_PATHS" => '"$(PODS_TARGET_SRCROOT)/ReactCommon" "$(PODS_TARGET_SRCROOT)" "$(PODS_ROOT)/RCT-Folly" "$(PODS_ROOT)/boost" "$(PODS_ROOT)/boost-for-react-native" "$(PODS_ROOT)/DoubleConversion" "$(PODS_ROOT)/Headers/Private/React-Core" "$(PODS_ROOT)/Headers/Private/Yoga"',
```
Comment (piaskowyk) [inline packages/react-native-worklets/RNWorklets.podspec:59]: Do we need a headers from Yoga in worklets lib too? 🤔

File: packages/react-native-worklets/android/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,60 @@
+project(Worklets)
```
Comment (piaskowyk) [inline packages/react-native-worklets/android/CMakeLists.txt:1]: `Worklets` of `worklets`?

**Review by piaskowyk — APPROVED** [review]

File: packages/react-native-worklets/RNWorklets.podspec (hunk 2 above)
Comment (tjzel) [reply packages/react-native-worklets/RNWorklets.podspec:59]: We'll clean this up once we are actually compiling things properly

File: packages/react-native-worklets/android/CMakeLists.txt (hunk 3 above)
Comment (tjzel) [reply packages/react-native-worklets/android/CMakeLists.txt:1]: What do you mean? `CMakeLists.txt` in reanimated directory specifies `Reanimated` project.

**Pull request merged**